core: disable rw triggers on replicas

ShelbyKS opened the pull request.

Commit:
NOTASK: disable triggers on replicas

Col-Waltz commented:
Thank you for the patch!

Please provide proper commit message for your changes:

```
core: disable rw triggers on replicas

<Some information about changes and implementation reason>
```

Commit:
core: disable rw triggers on replicas

after leader change in cluster there are ERR_READONLY errors on previous leader while processing on_disconnect triggers. I guess we can add early exit on replicas
